test/runtests.jl

```diff
@@ -4,13 +4,14 @@ using Test
 const testdir = dirname(@__FILE__)
 
 tests = [
-    "module_tests",
-    "utils/ElementFeatureUtils_tests",
-    "utils/GraphBuilding_tests",
-    "atoms/AtomGraph_tests",
-    "features/ElementFeature_tests",
-    "featurizations/GraphNodeFeaturization_tests",
-    "featurizedatoms_tests",
+    # "module_tests",
+    # "utils/ElementFeatureUtils_tests",
+    # "utils/GraphBuilding_tests",
+    # "atoms/AtomGraph_tests",
+    # "features/ElementFeature_tests",
+    "features/OrbitalFeature_tests",
+    # "featurizations/GraphNodeFeaturization_tests",
+    # "featurizedatoms_tests",
```

Comment:
I suspect this change was an accidental push, can we uncomment the rest of the tests please? 😆

Reply:
Oops, sorry about that. 😅